IPython/utils/text.py

```diff
@@ -517,7 +516,7 @@ def get_field(self, name, args, kwargs):
 # inside [], so EvalFormatter can handle slicing. Once we only support 3.4 and
 # above, it should be possible to remove FullEvalFormatter.
 
-@skip_doctest_py3
+@skip_doctest
```

Comment:
let's not skip doctest anymore and make the docstring python3 , even better could be a way to have that not a doctest.